daemon/daemon.go

```diff
@@ -136,6 +136,11 @@ func (daemon *Daemon) HasExperimental() bool {
 	return daemon.configStore != nil && daemon.configStore.Experimental
 }
 
+// GetFeatures returns the features map from configStore
+func (daemon *Daemon) GetFeatures() *map[string]bool {
```

Comment:
Better to name this `Features()` instead of `GetFeatures()`; golang convention is to not use a `Get` prefix for getters; https://golang.org/doc/effective_go.html#Getters

Reply:
Ah good call. 👍

Reply:
Fixed :)